src/shared_gui_components/MeasureManager.cpp

```diff
@@ -579,6 +579,10 @@ boost::optional<measure::OSArgument> MeasureManager::getArgument(const measure::
     Json::Value choiceValues = argument.get("choice_values", Json::Value(Json::arrayValue));
     Json::Value choiceDisplayNames = argument.get("choice_display_names", Json::Value(Json::arrayValue));
 
+    if (choiceValues.empty()) {
+      choiceValues = argument.get("choices_values", Json::Value(Json::arrayValue));
```

Comment:
This is a bug in the OS repo, there is a typo of `choices_values` instead of `choice_values`

Reply:
I fixed it already in https://github.com/NREL/OpenStudio/pull/5046 but it wasn't selected for 3.7.0 official...